app/controllers/concerns/curation_concerns/single_use_links_controller_behavior.rb

```diff
@@ -25,6 +25,16 @@ def create_show
       render text: curation_concerns.show_single_use_link_url(@su.downloadKey)
     end
 
+    def index
+      links = SingleUseLink.where(itemId: params[:id])
+      render partial: 'curation_concerns/file_sets/single_use_link_rows', locals: { single_use_links: links }
```

Comment:
@mjgiarlo you get `ActionView::MissingTemplate - Missing template curation_concerns/file_sets/single_use_link_rows` because it's expecting it to be a view template and not a partial since we'er in the `index` action.